src/command/DefaultMenus.js

```diff
@@ -277,8 +277,9 @@ define(function (require, exports, module) {
         editor_cmenu.addMenuDivider();
         editor_cmenu.addMenuItem(Commands.EDIT_CUT);
         editor_cmenu.addMenuItem(Commands.EDIT_COPY);
-        editor_cmenu.addMenuItem(Commands.EDIT_PASTE);
-
+        if(!(!window.Phoenix.browser.isTauri && window.Phoenix.browser.desktop.isFirefox)){
```

Comment:
`!window.Phoenix.browser.desktop.isFirefox || window.Phoenix.browser.isTauri` might be clearer to read.

Reply:
There should be one more place where the paste menu is there in the editor context menu: 
![image](https://user-images.githubusercontent.com/5336369/222964763-c33b8522-c666-4e16-9b54-5cf4277e458a.png)